src/makeRobot.js

```diff
@@ -38,7 +38,66 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goRight(step = 1) {
+      if (step > 0) {
+        this.coords.x += step;
+      }
+
+      return robot;
```

Comment:
```suggestion
    goRight(step = 1) {
      if (step > 0) {
        this.coords.x += step;
      }

      return this;
```

Можна повертати **this**, так як функцію ти викликаєш з robot`а.

Вбережеш себе від міскліку IMHO

Reply:
В інших трьох функціях аналогічно)